### PR TITLE
Add `table` option for `output-format` vulnerabilities in console

### PR DESCRIPTION
**Pull request by ken-chou-finn**
…running scan in blocking mode

Signed-off-by: Ken Chou <ken.chou@finn.ai>

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]
This would be very useful! I'm currently struggling with this.

**Comment by JAORMX** [comment]
This would close #168

File: index.js [hunk 1]
```diff
@@ -548,18 +557,21 @@ async function runScan({
...
+  if (exitCode > 0) {
     // There was a non-zero exit status but it wasn't because of failing severity, this must be
     // a grype problem
     if (!severityCutoff) {
       core.warning("grype had a non-zero exit status when running");
+    }
+
+    if (failBuild === true) {
+      if (showGrypeOutput) {
+        core.info(cmdOutput);
```
Comment (kzantow) [inline index.js:570]: I don't believe this is going to give a table-view output but only dumping the SARIF JSON, no?

**Comment by kzantow** [comment]
This is a great idea -- however, I think we could probably do this without a config parameter to just _always_ dump the table view to the console, WDYT?

**Comment by JAORMX** [comment]
> This is a great idea -- however, I think we could probably do this without a config parameter to just _always_ dump the table view to the console, WDYT?

to be honest, that behavior would be a nice default to have.

**Comment by JAORMX** [comment]
@kzantow is this a change that would need to go into grype itself or is it something that needs to happen in this action?

**Comment by jonasagx** [comment]
@ken-chou-finn thank for your work on this PR. On @JAORMX's [latest question](https://github.com/anchore/scan-action/pull/135#issuecomment-1108772134): grype currently prints the table and then errors out, like in the example below:

```
$ grype ubuntu:20.04 --fail-on medium                                                                                                                                                              
NAME          INSTALLED                 FIXED-IN            TYPE  VULNERABILITY   SEVERITY
coreutils     8.30-3ubuntu2                                 deb   CVE-2016-2781   Low
e2fsprogs     1.45.5-2ubuntu1                               deb   CVE-2022-1304   Medium
[other vulns removed for brevity of this snippet]
1 error occurred:
	* discovered vulnerabilities at or above the severity threshold
```

We just need to make use of the table output, so no need to change grype, just this action. I am happy to help with this PR and get the feature delivered.

**Comment by JAORMX** [comment]
@jonasagx that would be great!

**Comment by kzantow** [comment]
Sorry this has taken a while to get back to -- I think we may want to combine this with #187 by allowing users to specify a `format` parameter, which could be `table`.

One question: is there any situation a user wants to get _both_ a table output printed _and_ an output file of some sort?

**Pull request merged**